core/badge-urls/make-badge-url.js

```diff
@@ -62,7 +62,10 @@ function staticBadgeUrl({
   namedLogo,
   format = 'svg',
 }) {
-  const path = [label, message, color].map(encodeField).join('-')
+  const path = [label, message, color]
+    .map(s => s.toString())
```

Comment:
Could you move the coercion to the calling side instead? I feel like it's reasonable for this function to expect strings. It's really the web framework that is a bit unreasonable, about giving us different types depending on the input.

Reply:
Thanks! Done.